src/html/tags/HtmlTagSpan.js

```diff
@@ -1,24 +1,45 @@
 /* @flow */
 import React, { PureComponent } from 'react';
 import { View } from 'react-native';
+import { connect } from 'react-redux';
 
 import type { Actions, StyleObj } from '../../types';
+import { getOwnEmail } from '../../selectors';
 import renderHtmlChildren from '../renderHtmlChildren';
+import styles from '../HtmlStyles';
 
-export default class HtmlTagSpan extends PureComponent {
+class HtmlTagSpan extends PureComponent {
   props: {
     style?: StyleObj,
     actions?: Actions,
     cascadingStyle?: StyleObj,
+    attribs?: Object,
+    ownEmail: string,
```

Comment:
This is conceptually incorrect.
Our HtmlTagSpan should be aware only of general 'span' related stuff.
If we can not implement the styling by only generic code, we need to create a 'SpanMention' or something similar.